src/moma_expose.cpp

```diff
@@ -174,12 +174,13 @@ Rcpp::List cpp_multirank_BIC_grid_search(
     double EPS_inner,
     long MAX_ITER_inner,
     std::string solver,
-    int selection_criterion_alpha_u  = 0,  // 0 means grid, 1 means BIC search
-    int selection_criterion_alpha_v  = 0,
-    int selection_criterion_lambda_u = 0,
-    int selection_criterion_lambda_v = 0,
-    int max_bic_iter                 = 5,
-    int rank                         = 1)
+    int deflation_scheme       = 1,  // Defaults to 1 = PCA_Hotelling
+    int select_scheme_alpha_u  = 0,  // 0 means grid, 1 means BIC search
```

Comment:
This might be useful to `enum`-ify as well in case we add AIC / eBIC in the future.

Reply:
Please see commits "Enum-fy selection scheme encoding on R side", and "Enum-fy selection scheme encoding on C++ side".